src/core/server/rendering/views/template.tsx

```diff
@@ -211,9 +102,10 @@ export const Template: FunctionComponent<Props> = ({
 
         <link rel="manifest" href={favicon ? `` : `${uiPublicUrl}/favicons/manifest.json`} />
 
+        {/* Ignoring all of the above, Safari picks this one */}
         <link
           rel="mask-icon"
-          color="#e8488b"
```

Comment:
This made the favicon show pink on Safari.

Reply:
Are we able to just remove it and let the SVG provide the color?

Reply:
We probably should; the whole favicons part needs revamping.